src/main/java/net/glease/ggfab/DevEnvFixer.java

```diff
@@ -5,7 +5,7 @@
 public class DevEnvFixer implements Runnable {
     @Override
     public void run() {
-        ProcessingModSupport.aTGregSupport = true;
-        ProcessingModSupport.aEnableThermalFoundationMats = true;
+//        ProcessingModSupport.aTGregSupport = true;
```

Comment:
These don't seem to exist anymore....

Reply:
Just yeet the whole class. It's here solely to fix some material being missing in dev env.

Reply:
ok yeeted